src/element.ts

```diff
@@ -213,8 +220,9 @@ export abstract class Element {
   }
 
   /** Draw an element. */
-  // eslint-disable-next-line
-  abstract draw(...args: any[]): void;
+  draw(): void {
+    throw new RuntimeError('Element', 'Draw not defined');
```

Comment:
I notice Element is no longer an abstract class. That's fine.... but here the draw() method just throws a RuntimeError. So.... Element is still basically an abstract class that requires draw() to be overridden? 

Or should `draw()` have a basic implementation that just draws `this.text`?

Reply:
We do not have a x, y position to draw, this is why I left it without a basic call to renderText

Reply:
So if Element cannot be used by itself, should it be an abstract class (like it was in VexFlow 4)?

If you don't declare it as `abstract`, then a developer should be able to say new Element(...) and then start using it to draw elements on the staff, correct?